src/messages/mod.rs

```diff
@@ -375,7 +436,7 @@ impl SignedRoutingMessage {
         match &self.security_metadata {
             SecurityMetadata::None => !self.content.src.is_multiple(),
             SecurityMetadata::Partial(partial) => partial.shares.len() > partial.pk_set.threshold(),
-            SecurityMetadata::Full(_) => true,
+            SecurityMetadata::Full(_) | SecurityMetadata::Single(_) => true,
```

Comment:
do we need to have the same check (`!self.content.src.is_multiple()`) as in the `None` branch ?

Reply:
I am not sure.
It could have been a bug since user could sign a Authority::Section with a single signature, however, we do check that the signature is for the valid authority Client or ManagedNode. Moreover, a malicious node doing that could simply send that signed message directly, so there does not seem to be a need for any checks.

I guess the reason `SecurityMetadata::None => !self.content.src.is_multiple()` is because before we had a limitation where only Section messages were signed. I guess it could now be set to always false, but also there is an issue to address this properly everywhere: https://github.com/maidsafe/routing/issues/1756. So it does not seem we want to change `SecurityMetadata::None` here, and `SecurityMetadata::Single(_) => true` seem accurate too.